boards/hail/src/main.rs

```diff
@@ -35,15 +35,15 @@ unsafe fn load_processes() -> &'static mut [Option<kernel::process::Process<'sta
         static _sapps: u8;
     }
 
-    const NUM_PROCS: usize = 2;
+    const NUM_PROCS: usize = 4;
```

Comment:
Is there a reason we need NUM_PROCS at all instead of iterating until either out of memory or out of installed processes?